packages/big-design/src/components/Counter/spec.tsx

```diff
@@ -363,6 +367,26 @@ test('error shows when an array of Errors', () => {
   testIds.forEach((id) => expect(getByTestId(id)).toBeInTheDocument());
 });
 
+test('input value does not change when submitting the form via Enter key', async () => {
```

Comment:
Good point, I decided adding in here since in case in the future someone removes the "type="button" in the counter component, this test can fail? 👀  
I think I'm gonna update the `test` description a little bit might be confusing 😅